options:
  is_optitrack_on: true
  filter_gain: 0.2

